content/docs/csidriver/features/powerscale.md

```diff
@@ -287,6 +291,10 @@ spec:
     apiGroup: ""
 ```
 
+During the volume cloning process a read only snapshot of the source volume is made and a writable snapshot is created using the read only snapshot as the source. The use of snapshots provides way to quickly create snapshots and clones of source volumes. A SnapshotIQ license is required for snapshots to be created. Note that cloning of clones is not currently supported.

```

Comment:
Change to read "...snapshots provide a way to quickly..."

Reply:
Done.